pages/maps.tsx

```diff
@@ -347,9 +360,23 @@ const MapsGeneratorPage = () => {
           <Radio value="SZ_EVERY_OTHER">
             <Trans>SZ every other</Trans>
           </Radio>
+          <Radio value="CUSTOM_ORDER">
+            <Trans>Custom order</Trans>
+          </Radio>
         </Stack>
       </RadioGroup>
-      <FormLabel htmlFor="count" fontSize="sm">
+      <MultipleModeSelector
+        options={[
+          { label: "Splat Zones", value: "SZ", data: "SZ" },
+          { label: "Tower Control", value: "TC", data: "TC" },
+          { label: "Rainmaker", value: "RM", data: "RM" },
+          { label: "Clam Blitz", value: "CB", data: "CB" },
+        ]}
+        isDisabled={generationMode !== "CUSTOM_ORDER"}
+        setValue={getModeValues}
+        width={"90%"}
+      />
```

Comment:
This should only be rendered if Custom order is selected I think.

Reply:
Updated